test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -2784,6 +2784,10 @@ TEST_F(AtenXlaTensorTest, TestBernoulliScalarProb) {
     EXPECT_GT(frac, 0.06);
     EXPECT_LT(frac, 0.14);
   });
+
+  ExpectCounterNotChanged("aten::(?!_local_scalar_dense).*",
```

Comment:
What are we expecting here? 😄

Reply:
Like none of other aten:: is called? 
We called `item()` in test :P So this `_local_scalar_dense` need to be skipped.